### PR TITLE
Bemenu: 0.1.0 -> 0.3.0

### PR DESCRIPTION
**Pull request by NilsIrl**
###### Motivation for this change

New bemenu version.

###### Things done

Bump version. Use rec to not hard code commit.

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of ~~all~~ only wayland and ncurses support. X11 didn't work for me binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @thiagokokada

### CODE REVIEW

**Review by thiagokokada — APPROVED** [review]
:+1: 

I tested on all three backends (including x11).

Btw, I am not using this package anymore so if you want to adopt @NilsIrl, you're welcome, since it seems there is still some issues with this package like you reported here: https://github.com/NixOS/nixpkgs/pull/60661#issuecomment-569542225

**Comment by NilsIrl** [comment]
> there is still some issues with this package like you reported here: [#60661 (comment)](https://github.com/NixOS/nixpkgs/pull/60661#issuecomment-569542225)

For some reason I wasn't able to reproduce that. Might have had to do with `fonts.enableDefaultFonts` though that shouldn't have been a problem as `programs.sway.enable` was set to `true`.

**Comment by NilsIrl** [comment]
> so if you want to adopt @NilsIrl, you're welcome

It would be an honor, however, I have very limited knowledge of nix and bemenu so I don't think I would be the most capable for this.

File: pkgs/applications/misc/bemenu/default.nix
Comment (filalex77) [inline pkgs/applications/misc/bemenu/default.nix]:
```suggestion
    rev = version;
```

File: pkgs/applications/misc/bemenu/default.nix
Comment (filalex77) [inline pkgs/applications/misc/bemenu/default.nix]:
```suggestion
    repo = pname;
```

**Review by Br1ght0ne — COMMENTED** [review]
Other than a few nitpicks, LGTM.

```
[3 built, 69 copied (526.5 MiB), 118.7 MiB DL]
https://github.com/NixOS/nixpkgs/pull/76676
1 package built:
bemenu
```

**Review by Br1ght0ne — APPROVED** [review]
LGTM.

To maintainers: please squash commits before merging.

**Review by thiagokokada — APPROVED** [review]

**Comment by Infinisil** [comment]
I'd appreciate for PR authors to squash the commits themselves because:
- If the change doesn't make sense as a single commit, the GitHub squash button can't be used
- It maintains the original SHA1 and signature of the PR author
- It requires people to learn how to interactively `git rebase` and how to [name their commits](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes)

**Comment by NilsIrl** [comment]
Ok, rebased it. Only took be 4 tries to get @filalex77 to be recognised as co-author (I wasn't sure where to put the line in the commit message)

**Pull request merged**
